lib/OpenQA/WebAPI/Auth/OpenID.pm

```diff
@@ -82,7 +82,6 @@ sub auth_login {
 sub auth_response {
     my ($self) = @_;
 
-    # FIXME: Mojo6 hack, remove after version bump
```

Comment:
the opposite is true. this is code to work for both 5 and 6 - as we require 6 now anyway, we can clean this up. That's what these FIXMEs are about